server/crashmanager/templates/bugzilla/list.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'layouts/layout_base.html' %}
+
+{% block body_content %}
+<div class="panel panel-default">
+    <div class="panel-heading"><i class="glyphicon glyphicon-list-alt"></i> Templates</div>
+    <div class="panel-body">
+        <div>
+            <a href="{% url 'crashmanager:templatecreatebug' %}" class="btn btn-primary">
+                <i class="glyphicon glyphicon-plus"></i> Add bug template
+            </a>
+            <a href="{% url 'crashmanager:templatecreatecomment' %}" class="btn btn-primary">
+                <i class="glyphicon glyphicon-plus"></i> Add comment template
+            </a>
+        </div>
+        <div class="pagination">
+            <span class="step-links">
+                {% if page_obj.has_previous %}
+                    <a title="Previous" href="?page={{ page_obj.previous_page_number }}" class="glyphicon glyphicon-chevron-left"></a>
+                {% endif %}
+                <span class="current">Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}.</span>
+                {% if page_obj.has_next %}
+                    <a title="Next" href="?page={{ page_obj.next_page_number }}" data-placement="top" class="glyphicon glyphicon-chevron-right"></a>
+                {% endif %}
+            </span>
+        </div>
+        <table class="table table-condensed table-hover table-bordered table-db">
+            <thead>
+            <tr>
+                <th style="width: 20px;">ID</th>
+                <th style="width: 40px;">Name</th>
+                <th style="width: 30px;">Mode</th>
```

Comment:
Could we change `Mode` to `Type`? I think it is easier for a user to think that the template "has this type" instead of it "is in this mode". (only here where it is displayed, `.mode` in the model is fine)

Reply:
Sure, no problem!